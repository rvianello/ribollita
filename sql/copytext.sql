CREATE EXTENSION IF NOT EXISTS ribollita;

SELECT copytext('Hello World!');
