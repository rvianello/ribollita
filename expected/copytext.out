CREATE EXTENSION IF NOT EXISTS ribollita;
SELECT copytext('Hello World!');
   copytext   
--------------
 Hello World!
(1 row)

